hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/metrics2/sink/PrometheusMetricsSink.java

```diff
@@ -129,6 +142,12 @@ public void writeMetrics(Writer writer) throws IOException {
             sep = ",";
           }
         }
+        if (customTags.size() > 0) {
```

Comment:
Nit: I think it is clearer to use `#isEmpty()`.
```suggestion
        if (!customTags.isEmpty()) {
```